### PR TITLE
Ft/references

### PR DESCRIPTION
**Pull request by nsengaqueen**
In this PR we added the following :

- [ ] We created a new reference branch
- [ ] Deleted the unnecessary files in our branch (Hero, Contact, About, Services)
- [ ] Made the page responsive
- [ ] Reducing the non-arbitrary values where possible

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```
Comment (Algor-Fernand) [inline .vscode/settings.json:2]: I am not sure if we need this in our project.

File: src/sections/References.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<section class="flex flex-col items-end gap-0 pt-20 pb-20 pl-2 pr-16 text-2xl md:gap-3 lg:gap-3 bg-customYellow md:p-32 lg:p-20 rounded-b-md font-outfit gap pt">
```
Comment (Algor-Fernand) [inline src/sections/References.html:1]: I think if padding top and padding bottom have the same padding you can change into a single class: 'py'.

File: tailwind.config.js [hunk 3]
```diff
@@ -2,13 +2,22 @@
...
     },
-
   },
-  plugins: [],
-}
-
+  plugins: [
+    function ({ addUtilities }) {
+      addUtilities({
+        '.clip-bottom': {
```
Comment (Algor-Fernand) [inline tailwind.config.js:17]: I love this 🤩🤩🤩

File: .vscode/settings.json
Comment (uwumukiza123) [inline .vscode/settings.json:1]: Are you using this Json file? If not you can remove it since is empty

File: src/sections/References.html
Comment (uwumukiza123) [inline src/sections/References.html:1]: For this references It's better to put them in one array for better performance and fewer lines of code

**Review by uwumukiza123 — CHANGES_REQUESTED** [review]

**Review by Algor-Fernand — APPROVED** [review]

**Pull request merged**